Views/AbstractViews/AbstractApiView.php

```diff
@@ -0,0 +1,89 @@
+<?php
+declare(strict_types=1);
+
+namespace Views\AbstractViews;
+
+use Models\AbstractProjectModels\Session\User\AbstractSessionModel as UserSessionModel;
+
+abstract class AbstractApiView extends AbstractDefaultView
```

Comment:
Do we need to keep it as abstract class? i can see you have 3 classes that just extends it with the same code. And no one of them has any specific logic. You can create it as basic class and use this class whenever you need